valohai_yaml/objs/pipelines/override.py

```diff
@@ -0,0 +1,57 @@
+from collections import OrderedDict
+from typing import Iterable, List, Optional, Union
+
+from valohai_yaml.objs import Mount, Parameter
+from valohai_yaml.objs.base import Item
+from valohai_yaml.objs.environment_variable import EnvironmentVariable
+from valohai_yaml.objs.input import Input
+from valohai_yaml.objs.utils import check_type_and_dictify, check_type_and_listify, serialize_into
+from valohai_yaml.types import SerializedDict
+from valohai_yaml.utils.parse import parse_common_kwargs
+
+
+class Override(Item):
+    """Represents a Override class within an Execution Node."""
+
+    def __init__(
+        self,
+        *,
+        image: Optional[str] = '',
+        command: Optional[Union[List[str], str]] = '',
+        parameters: Optional[Iterable[Parameter]] = (),
+        inputs: Optional[Iterable[Input]] = (),
+        mounts: Optional[Iterable[Mount]] = (),
+        environment_variables: Optional[Iterable[EnvironmentVariable]] = (),
+        environment: Optional[str] = None,
+    ) -> None:
+        self.image = image
+        self.command = command
+        self.environment = (str(environment) if environment else None)
+        self.mounts = check_type_and_listify(mounts, Mount)
+        self.inputs = check_type_and_dictify(inputs, Input, 'name')
+        self.parameters = check_type_and_dictify(parameters, Parameter, 'name')
+        self.environment_variables = check_type_and_dictify(environment_variables, EnvironmentVariable, 'name')
+
+    @classmethod
+    def parse(cls, data: SerializedDict) -> 'Override':
+        kwargs = parse_common_kwargs(data)
+        inst = cls(**kwargs)
+        inst._original_data = data
+        return inst
+
+
+
+    def serialize(self) -> OrderedDict:  # type: ignore[type-arg]
```

Comment:
Similarly, this should take care to only serialize the values that are set (not `None`).